app/views/project_content_items/_add_flags.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class='row flag-for-review'>
+  <h1>Flag '<%= content_item.title %>' for review</h1>
+
+  <%= form_for content_item, url: update_flags_project_content_item_path(project, content_item), method: :post, html: { class: 'col-md-6', data: { module: 'radio-toggle' } } do |f| %>
+
+    <div class='radio'>
+      <label>
+        <%= f.radio_button :flag, "needs_help" %>
+        I need help tagging this
+      </label>
+    </div>
+
+    <div class='radio-toggle'>
```

Comment:
This class seems redundant? I can't seem to find any CSS for it and the [RadioToggle module](https://github.com/alphagov/govuk_admin_template/blob/master/app/assets/javascripts/govuk-admin-template/modules/radio_toggle.js) doesn't use this class. Was it added just to group the two child `div`s or signify that this is where the JS module is being used?

Reply:
yep, this is entirely vestigal I think. good :eyes: